pretyping/reductionops.mli

```diff
@@ -58,7 +58,8 @@ module Stack : sig
 
   val pr_app_node : (EConstr.t -> Pp.t) -> app_node -> Pp.t
 
-  type case_stk
+  type case_stk =
+    case_info * EInstance.t * EConstr.t array * EConstr.t pcase_return * EConstr.t pcase_invert * EConstr.t pcase_branch array
```

Comment:
It seems you only build values of this type. If that's the case (no pun intended) don't expose the type definition but a builder function instead.

Reply:
It is the case... Done

Reply:
You didn't hide the type though.

Reply:
Lol, looks like I was a bit absent-minded. Done